src/utils/path.ts

```diff
@@ -0,0 +1,11 @@
+export const isWindows = typeof process !== 'undefined' && process.platform === 'win32';
+
+const backslashRegex = /\\/g;
+
+function slash(path: string): string {
+  return path.replace(backslashRegex, '/');
+}
+
+export function normalizePath(id: string): string {
+  return require('node:path').posix.normalize(isWindows ? slash(id) : id);
```

Comment:
this shouldn't work since we're in ES modules

but i'm not sure you need to care about `isWindows`. you can probably just replace the slashes either way:

```ts
import path from 'node:path';

export function normalizePath(id: string): string {
  return id.replaceAll(backslashRegex, '/');
}
```

`normalize` just replaces repeated slashes and resolves `..` and `.` segments. so im not sure if its giving us anything here really